README.md

```diff
@@ -94,7 +94,7 @@ debugpy.wait_for_client()  # optional
 
 Or alternativley, prefix your commands with `python -m debugpy --listen 0.0.0.0:5678`.
 ```shell
-docker-compose run app python -m debugpy --listen 0.0.0.0:5678 manage.py test
+docker-compose run app -p 5678:5678 python -m debugpy --listen 0.0.0.0:5678 manage.py test
```

Comment:
if using exec instead of run we wouldn't need to manually specify the port

Reply:
Since I expose them from `docker-compose.local.yml` I guess this is extra that we don't need anymore.

Reply:
Wait, second though, why we need to run `exec`?

Reply:
exec runs in the already running container with already open ports, while run will try to start another container (which I guess should fail due to already forwarded ports ?)

Reply:
> Since I expose them from `docker-compose.local.yml` I guess this is extra that we don't need anymore.

Hm, this does now work for whatever reason??? In docker compose I have `ports: ["5679:5679"]` in `docker-compose.override.local.yml` and I call:
```
docker-compose run worker_wrapper python -m debugpy --listen 0.0.0.0:5679 manage.py dequeue
``` 
and I cannot connect from VS code for whatever reason. If I remove `ports` from the compose definition and run:
```
docker-compose run -p 5679:5679 worker_wrapper python -m debugpy --listen 0.0.0.0:5679 manage.py dequeue
```
however it works? Ideas?

Reply:
Ah, as discussed on chat, when using `docker-compose run`, the ports are ignored so there is no conflict with already running containers, unless `--service-ports` is passed.

So finally what I would do:

- keep the ports in the docker compose config, so you can use `docker-compose exec manage.py tests`;
- recommend using `docker run -p 5678:5678 manage.py tests` in the docs;